### ISSUE TITLE
Lifetime problem

### ISSUE THREAD

**Issue by daniiarov-alym** [ArtHome12/Demi#35]
On Rust 1.64.0 the project is not being built because of lifetime problem:

```
error[E0597]: `desc_set` does not live long enough
     |
1529 |                     set: desc_set.raw_mut(),
     |                          ^^^^^^^^^^^^^^^^^^ borrowed value does not live long enough
...
1549 |     }
     |     -
     |     |
     |     `desc_set` dropped here while still borrowed
     |     borrow might be used here, when `write_map` is dropped and runs the `Drop` code for type `BTreeMap`

```

**Comment by daniiarov-alym** [comment]
Problem is caused by wgpu_core 0.7.1 package

**Comment by ArtHome12** [comment]
The problem must be fixed f3dc10c
